lib/main/atom/views/symbols/highlightComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import * as etch from "etch"
+import {QueryMatch} from "./match.d"
+
+export interface Props extends JSX.Props {
+  matches: QueryMatch[]
+  styleClass?: string
+}
+
+export class HighlightComponent implements JSX.ElementClass {
+  constructor(public props: Props) {
+    etch.initialize(this)
+  }
+
+  public async update(props: Partial<Props>) {
+    this.props = {...this.props, ...props}
+    await etch.update(this)
+  }
+
+  public async destroy() {
+    await etch.destroy(this)
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div class={this.props.styleClass || ""}>
```

Comment:
As mentioned above, probably a better idea to render this as an inline element, so `span` instead of `div` seems reasonable.

Also, while here, setting `class` property to `""` will create an empty `class` attribute on the DOM element. Setting `class` property to `undefined` will not create `class` attribute at all.

While at it, might as well rename `styleClass` to just `class` for the sake of consistency. The trick is to use quotes in the type definition.